extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -25,6 +25,15 @@ import { DeployDatabaseDialog } from '../dialogs/deployDatabaseDialog';
 import { NetCoreTool, DotNetCommandOptions } from '../tools/netcoreTool';
 import { BuildHelper } from '../tools/buildHelper';
 
+export enum ExtractTarget {
+	dacpac = 0,
```

Comment:
Why are you using numbers for this? If you used strings it seems like it'd be much easier. And is there a reason you wouldn't just capitalize the values here instead of doing it in the quick-pick?

Reply:
also, is this needed? Looks like mssql.ExtractTarget is used in this file

Reply:
Yeah it's still needed because of how she's populating the quickpick dropdown. Typings files such as mssql.d.ts don't exist at runtime so you can't query the key values of an enum like this. The workaround is to define it in multiple places like this.

You see something similar in our azdata/vscode stuff. Enums defined there need to also have their implementation defined elsewhere (like sqlExtHost.api.impl.ts) for runtime dependencies.

Reply:
As for capitalizing, it was just to match the other enums' format in the file where I added it initially. I will have the string enum usage as a TODO here.